ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -494,6 +494,32 @@ tests:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-ovn-masterscale
+  optional: true
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      ROTATE_CLOUD_CREDENTIALS: "true"
+      SCALE_MASTERS_FLAVOR_ALTERNATE: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    workflow: openshift-e2e-openstack-ipi
+  timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-ovn-masterscale-zones
+  optional: true
+  steps:
+    cluster_profile: openstack-kuryr
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: minimal
+      ROTATE_CLOUD_CREDENTIALS: "true"
```

Comment:
I don't know why you set `ROTATE_CLOUD_CREDENTIALS` to True. Because of that I had to do https://github.com/openshift/release/pull/41437 because the steps can't work well behind a proxy (the case for jobs running in mecha)

Reply:
is it because the job is going to take long time?

Reply:
We want to enable cloud credentials rotation in all jobs that support app creds, because it's cheap and ensures all operator react correctly to creds rotation. For example, we've recently discovered issues with the manila csi operator.